paymentsheet/src/main/java/com/stripe/android/lpmfoundations/AddPaymentMethodUiDefinition.kt

```diff
@@ -0,0 +1,55 @@
+package com.stripe.android.lpmfoundations
+
+import com.stripe.android.paymentsheet.PaymentSheet
+
+/**
+ * Encapsulates the information to show the information required create the payment method.
+ */
+internal data class AddPaymentMethodUiDefinition(
+    /**
+     * Identifier used for saving and restoring.
+     */
+    val identifier: String,
+
+    /**
+     * The information required to show the selection UI.
+     */
+    val addPaymentMethodSelectorUiDefinition: AddPaymentMethodSelectorUiDefinition,
```

Comment:
Somehow I'd expect the `InitialAddPaymentMethodState` to own a `AddPaymentMethodSelectorUiDefinition`